trace-sdk/src/test/java/io/bitrise/trace/data/collector/network/okhttp/OkHttpDataListenerTest.java

```diff
@@ -169,4 +179,41 @@ public void createNetworkData_ShouldCreateCorrectNetworkData() {
                                                                   .setStart(DUMMY_START_TIME).setEnd(DUMMY_END_TIME);
         assertThat(actualValue, is(expectedValue));
     }
+
+    @Test
+    public void constructor() {
+        final OkHttpDataListener listener = new OkHttpDataListener(mockContext);
+        assertNotNull(listener);
```

Comment:
I do not think constructor calls can return null.

Reply:
maybe this was an eager test - will remove.